app/src/main/res/layout/topic_practice_skill_view.xml

```diff
@@ -16,11 +16,10 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    app:barrierDirection="right"
-    app:constraint_referenced_ids="button_example,text_view_status">
```

Comment:
@rt4914,The line `    app:constraint_referenced_ids="button_example,text_view_status"` is deleted because the line was giving error as the ids mentioned in the line do not exist anymore in the file. App was running successfully without removing the line. But because of error, I removed it.

Reply:
Undo this change, we need these lines of code.

Reply:
Please reply to each and every comment/conversation.